docs/Serverless.md

```diff
@@ -0,0 +1,99 @@
+<h1 align="center">Serverless</h1>
+
+Run serverless applications and REST APIs using your existing Fastify application.
+
+> Fastify is not designed to run on the AWS Lambda platform.
```

Comment:
Using the quote-style formatting is not accurately representing the importance of this message. I'd recommend making this normal text and even including an `<strong>Attention Readers:</strong>` to the beginning of the paragraph.

Reply:
Technically, we normally do callouts like so:

> ### Note
> This is a note

Which is:

```
> ### Note
> This is a note
```

Reply:
Ahh okay this would be a good comprise! As long as we can callout the reader's attention

Reply:
✅